src/frontend/Komponenter/Behandling/Totrinnskontroll/SendTilBeslutterFooterVerge.tsx

```diff
@@ -0,0 +1,141 @@
+import * as React from 'react';
+import { ChangeEvent, useState } from 'react';
+import { IPersonopplysninger, IVergemål } from '../../../App/typer/personopplysninger';
+import UIModalWrapper from '../../../Felles/Modal/UIModalWrapper';
+import { AlertStripeFeil } from 'nav-frontend-alertstriper';
+import { Hovedknapp, Knapp } from 'nav-frontend-knapper';
+import { Footer, MidtstiltInnhold, StyledHovedknapp } from './SendTilBeslutterFooter';
+import { Checkbox, CheckboxGruppe } from 'nav-frontend-skjema';
+import { useApp } from '../../../App/context/AppContext';
+import { RessursFeilet, RessursStatus, RessursSuksess } from '../../../App/typer/ressurs';
+import { ModalAction, ModalType, useModal } from '../../../App/context/ModalContext';
+import { useBehandling } from '../../../App/context/BehandlingContext';
+import { EBrevmottakerRolle, IBrevmottaker } from './typer';
+
+export const SendTilBeslutterFooterVerge: React.FC<{
+    behandlingId: string;
+    personopplysninger: IPersonopplysninger;
+}> = ({ personopplysninger, behandlingId }) => {
+    const [visModal, settVisModal] = useState<boolean>(false);
+    const [feilmelding, settFeilmelding] = useState<string>('');
+    const [laster, settLaster] = useState(false);
+    const [valgteMottakere, setteValgteMottaker] = useState<IBrevmottaker[]>([]);
+    const { axiosRequest } = useApp();
+    const { modalDispatch } = useModal();
+    const { hentTotrinnskontroll } = useBehandling();
+
+    const lukkModal = () => {
+        settVisModal(false);
+        settFeilmelding('');
+    };
+
+    const sendTilBeslutterMedMottakere = () => {
+        settLaster(true);
+        settFeilmelding('');
+        axiosRequest<string, IBrevmottaker[]>({
+            method: 'POST',
+            url: `/familie-ef-sak/api/vedtak/${behandlingId}/send-til-beslutter/verge`,
+            data: valgteMottakere,
+        })
+            .then((res: RessursSuksess<string> | RessursFeilet) => {
+                if (res.status === RessursStatus.SUKSESS) {
+                    lukkModal();
+                    hentTotrinnskontroll.rerun();
+
+                    modalDispatch({
+                        type: ModalAction.VIS_MODAL,
+                        modalType: ModalType.SENDT_TIL_BESLUTTER,
+                    });
+                } else {
+                    settFeilmelding(res.frontendFeilmelding);
+                }
+            })
+            .finally(() => settLaster(false));
+    };
+
+    const bruker: IBrevmottaker = {
+        navn: personopplysninger.navn.visningsnavn,
+        personIdent: personopplysninger.personIdent,
+        mottakerRolle: EBrevmottakerRolle.BRUKER,
+    };
+    const verger = personopplysninger.vergemål
+        .filter((vergemål, indeks, self) => self.indexOf(vergemål) === indeks)
```

Comment:
Hva skjer her? 🤯

Reply:
Tanken var å filtrerer bort duplikater 😇  
En person kan kan ha flere vergemål for samme bruker, så vergemål listen kan potensielt sett inneholde samme person flere ganger. Men ser nå at dette ikke funker, fordi vergemålstypene vil være forskjellige...

Fikser.

Reply:
:D Bra jeg spurte - vurder å lag en funksjon - `filtrerBortDuplikateVerger` eller noe i den duren :D